src/components/form/ha-form-float.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  customElement,
+  LitElement,
+  html,
+  property,
+  TemplateResult,
+  query,
+} from "lit-element";
+import { HaFormElement, HaFormSchema } from "./ha-form";
+import { fireEvent } from "../../common/dom/fire_event";
+
+import "@polymer/paper-input/paper-input";
+// Not duplicate, is for typing
+// tslint:disable-next-line
+import { PaperInputElement } from "@polymer/paper-input/paper-input";
+
+@customElement("ha-form-float")
+export class HaFormFloat extends LitElement implements HaFormElement {
+  @property() public schema!: HaFormSchema;
```

Comment:
I think that we can improve our typing if we have a schema for `HaFormFloatSchema` etc. `HaFormSchema = HaFormBooleanSchema | HaFormFloatSchema`